src/doc/dev_manual/LiveIncidentResponse.rst

```diff
@@ -0,0 +1,121 @@
+directly support on systems where we have accounts, hosted binary downloads for
+common platforms and the tools to build from sources.
+
+The practice of having software development staff *integrated* with *operations*
+is more commonly referred to as *DevOps*. There is a pretty good
+`video <https://youtu.be/XoXeHdN2Ayc>`_ that introduces these concepts.
+
+This document explains our procedures for how VisIt_ software developers handle
+active customer support calls. Some of the goals of this proccess are...
+
+  * To try to *load balance* the work of incident response in an equitable way
+    across the development team.
+  * To achieve a 1-hour response time.
+  * To reduce incident response interruptions for the team as a whole.
+
+For the VisIt_ project, normal business hours are 8am-12pm and 1-5pm, Monday
+through Friday excluding LLNL holidays. The work of *live* customer support is
+allocated in one-week intervals. During a week, one developer is assigned
+as the *primary* customer support contact and a second developer is assigned
+as a *backup*. Live customer support issues will be logged and tracked in a
+separate GitHub respository within the
+`visit-dav GitHub organization. <https://github.com/visit-dav>`_
+
+
+The Primary's Role
+------------------
+
+The primary's role is to answer all incidents that occur during that week
+including those that came in during the preceding weekend/holiday. Other
+developers who may be contacted directly by customers should redirect such
+inquires to the primary. The primary's goal is to *wrap-up* all incidents
+by the end of their week. If an issue cannot be wrapped-up, it gets handed-off
+to next week's primary. Handoff's of incidents are managed formally with
+an email to customer and the next week's primary and backup contact information.
+
+For each new incident, the primary should file an issue ticket in the
+`visit-live-incidents <https://github.com/visit-dav/visit-live-incidents/issues>`_
+repo and assign themselves. When the incident is resolved, the associated issue
+should be closed. Primary should endeavour to capture all relevant information
+and communications involving the incident in this issue. For users who do not have
+GitHub accounts, this will require the primary to do a modest amount of cutting
+and pasting between email and GitHub issues.
+
+The goal of the primary is to reach a *resolution* of each incident. Resolution
+often involves one or more of the following activities.
+
+  * Answering a question or refering user to documentation.
+  * Determining if user's issue is known (e.g. an issue ticket already exists).
+  * Updating a known issue with additional information from this user, perhaps
+    reprioritizing the issue or putting the ticket back into the UN-reviewed
+    state for further discussion at a VisIt_ project meeting.
+  * Identifying and subsequently fixing a *low-hanging fruit* issue.
+  * Developing a reproducer for developers.
```

Comment:
Not sure what this means...

Reply:
A *reproducer* is the data, configuration, steps involved to reproduce the issue.